packages/expo-audio/src/AudioModule.types.ts

```diff
@@ -27,18 +94,86 @@ export declare class AudioPlayer {
    */
   pause(): void;
 
-  prepareToPlay(): void;
+  /**
+   * Seeks the playback by the given number of seconds.
+   * @param seconds The number of seconds to seek by.
+   */
+  seekTo(seconds: number): Promise<void>;
 
-  isLoopingEnabled(enabled: boolean): void;
+  /**
+   * Sets the current playback rate of the audio.
+   * @param rate The playback rate of the audio.
+   * @param pitchCorrectionQuality The quality of the pitch correction.
+   */
+  setPlaybackRate(second: number, pitchCorrectionQuality?: PitchCorrectionQuality): void;
 
-  setRate(rate: number): void;
   /**
-   * Seeks the playback by the given number of seconds.
+   * Release the player and frees up resources.
+   */
+  release(): void;
+}
+
+export interface AudioRecorder {
```

Comment:
If something is a class on the native side, we should rather use `declare class AudioRecorder { ... }`